### PR TITLE
Docker Desktop FAQ updates

### PR DESCRIPTION
**Pull request by usha-mandya**
Updates to Docker Desktop Windows and Mac FAQs

### CODE REVIEW

File: docker-for-windows/faqs.md [hunk 1]
```diff
@@ -239,13 +146,11 @@ Getting Started topic. For more about Docker Machine itself, see
...
 
-## Windows Requirements
+## Windows requirements
 
 ### How do I run Windows containers on Docker Desktop on Windows Server?
 
-See [About Windows containers and Windows Server](/install/windows/docker-ee/#about-docker-ee-containers-and-windows-server).
-
-A full tutorial is available in [docker/labs](https://github.com/docker/labs){: target="_blank" class="_"} at
+A tutorial about running Windows containers is available in [docker/labs](https://github.com/docker/labs){: target="_blank" class="_"} at
```
Comment (usha-mandya) [inline docker-for-windows/faqs.md]: You can find a tutorial about running Windows containers on Windows Server in [Getting Started with Windows Containers](https://github.com/docker/labs/blob/master/windows/windows-containers/README.md){: target="_blank" class="_"}.

Comment (usha-mandya) [reply docker-for-windows/faqs.md]: Fixed.

File: docker-for-windows/faqs.md [hunk 2]
```diff
@@ -137,7 +49,23 @@ On Docker Desktop, clients can connect to the Docker Engine through a
 This sets `DOCKER_HOST` and `DOCKER_CERT_PATH` environment variables to the
 given values (for the named pipe or TCP socket, whichever you use).
 
-See also [Docker Engine API](/engine/api) and the Docker Desktop for Windows forums topic [How to find the remote API](https://forums.docker.com/t/how-to-find-the-remote-api/20988){: target="_blank" class="_"}.
+See also [Docker Engine API](/engine/api.md) and the Docker Desktop for Windows forums topic [How to find the remote API](https://forums.docker.com/t/how-to-find-the-remote-api/20988){: target="_blank" class="_"}.
+
+### How do I connect from a container to a service on the host?
+
+Windows has a changing IP address (or none if you have no network access). We recommend that you connect to the special DNS name `host.docker.internal`, which resolves to the internal IP address used by the host. This is for development purpose and will not work in a production environment outside of Docker Desktop for Windows.
```
Comment (djs55) [inline docker-for-windows/faqs.md]: minor: maybe "purposes"?

Comment (usha-mandya) [reply docker-for-windows/faqs.md]: Fixed now. Thanks.

File: docker-for-mac/faqs.md [hunk 3]
```diff
@@ -175,11 +107,54 @@ VirtualBox or VMWare Fusion.
 
 ### What is the benefit of HyperKit?
 
-It is thinner than VirtualBox and VMWare fusion, and the version we include is
-tailor made for Docker workloads on the Mac.
+HyperKit is thinner than VirtualBox and VMWare fusion, and the version we include is tailor made for Docker workloads on the Mac.
```
Comment (traci-morrison) [inline docker-for-mac/faqs.md]: s/tailor made/tailor-made

Do you think this word could be confusing to non-native English speakers?

File: docker-for-mac/faqs.md [hunk 4]
```diff
@@ -175,11 +107,54 @@ VirtualBox or VMWare Fusion.
...
+### How can I opt out of sending my usage data?
+
+If you do not want to send of usage data, use the Stable channel. For more
+information, see [What is the difference between the Stable and Edge versions of Docker Desktop](#stable-and-edge-channels).
+
+### How is personal data handled in Docker Desktop?
+
+When uploading diagnostics to help Docker with investigating issues, the
+uploaded diagnostics bundle may contain personal data such as usernames and IP
+addresses. The diagnostics bundles are only accessible to Docker Inc. employees
```
Comment (traci-morrison) [inline docker-for-mac/faqs.md]: s/Docker Inc./Docker, Inc.

File: docker-for-mac/faqs.md [hunk 5]
```diff
@@ -175,11 +107,54 @@ VirtualBox or VMWare Fusion.
...
+who are directly involved in diagnosing Docker Desktop issues. By default Docker
```
Comment (traci-morrison) [inline docker-for-mac/faqs.md]: s/By default/By default,
S/Docker Inc./Docker, Inc.

File: docker-for-mac/faqs.md [hunk 6]
```diff
@@ -175,11 +107,54 @@ VirtualBox or VMWare Fusion.
...
+Inc. will delete uploaded diagnostics bundles after 30 days unless they are
+referenced in an open issue on the
+[docker/for-mac](https://github.com/docker/for-mac/issues) or
+[docker/for-win](https://github.com/docker/for-win/issues) issue trackers. If an
+issue is closed, Docker Inc. will remove the referenced diagnostics bundles
+within 30 days. You may also request the removal of a diagnostics bundle by
+either specifying the diagnostics ID or via your GitHub ID (if the diagnostics
+ID is mentioned in a GitHub issue). Docker Inc. will only use the data in the
+diagnostics bundle to investigate specific user issues, but may derive high
+level (non personal) metrics such as the rate of issues from it.
```
Comment (traci-morrison) [inline docker-for-mac/faqs.md]: s/high level/high-level

**Comment by usha-mandya** [comment]
Thanks @traci-morrison. I've fixed the review comments. Could you PTAL?

File: docker-for-mac/faqs.md (hunk 3 above)
Comment (usha-mandya) [reply docker-for-mac/faqs.md]: Good point. I've updated to 'customized' for clarity.

File: docker-for-mac/faqs.md (hunk 4 above)
Comment (usha-mandya) [reply docker-for-mac/faqs.md]: Fixed this and all other occurrences in this topic and also in the Windows topic.

File: docker-for-mac/faqs.md (hunk 5 above)
Comment (usha-mandya) [reply docker-for-mac/faqs.md]: Fixed.

File: docker-for-mac/faqs.md (hunk 6 above)
Comment (usha-mandya) [reply docker-for-mac/faqs.md]: Fixed.

**Pull request merged**